pkg/BUILD

```diff
@@ -89,3 +91,12 @@ py_library(
     srcs_version = "PY2AND3",
     visibility = ["//visibility:public"],
 )
+
+# Used within experimental/tests/ for external repo genpkg tests
```

Comment:
This broke things last time I merged it.
It seems unused, and there is no file testdata/loremipsum.txt. It is in tests/testdata.

Reply:
Yeah, I have no idea how this got in here.  It will be removed when this PR is updated.